srcpkgs/vscode/template

```diff
@@ -11,60 +11,60 @@ maintainer="shizonic <realtiaz@gmail.com>"
 license="MIT"
 homepage="https://code.visualstudio.com/"
 distfiles="https://github.com/Microsoft/vscode/archive/${version}.tar.gz"
-checksum=3f0cde58ab558cd227c61900222f4363589a81e4df2fac2bb2042229c90a7ee8
+checksum=c3c7f10c19b76465a63c6179762187e53ddf1f3cee85f82b2ec447c9b6e9407a
 
 if [ "$XBPS_TARGET_WORDSIZE" = "32" ]; then
-	broken="FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory"
+    broken="FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory"
```

Comment:
Please revert indentation change, it should be tab.

Reply:
And really in all other places, I think your editor had a field day :p

Reply:
Just did it. The new version turned the autoformatter on by default, it seems.